.travis.yml

```diff
@@ -4,4 +4,10 @@ rvm:
   - 2.2.2
   - 1.9.3
   - jruby
+gemfile:
+  - gemfiles/Gemfile.rails-3.2
```

Comment:
First I put Gemfiles in spec folder, but I didn't think about bundler_args so I didn't work, thanks, nice to know.

Do you mind to fix this and other notes about code style by yourself and merge it? I think you know better how it should looks like